README.md

```diff
@@ -108,18 +106,12 @@ Use to inspect all session variables. Useful to  dump _path_ variables like `$fi
 
 ## `omf destroy`
 
-Uninstall Oh My Fish. See [uninstall](#uninstall) for more information.
+Uninstall Oh My Fish.
 
 # :triangular_flag_on_post: Advanced
 + [Startup](#startup)
 + [Dotfiles](#dotfiles)
 + [Core Library](#core-library)
-+ [Packages](#packages)
-  + [Creating](#creating)
-  + [Submitting](#submitting)
-  + [Initialization](#initialization)
-  + [Uninstall](#uninstall)
-  + [Ignoring](#ignoring)
 
 ## Startup
```

Comment:
We shouldn't advise people to change `config.fish` but to add customisation to the `init.fish` file or to create a custom plugin.